### PR TITLE
Issue #SB-13171 fix: device id will be fetched from local storage if exists

### PR DESCRIPTION
**Pull request by sudip-mukherjee**

### CODE REVIEW

File: js/core/telemetryV3Interface.js [hunk 1]
```diff
@@ -556,21 +556,27 @@ var Telemetry = (function() {
...
         NOT_AVAILABLE: 'not available',
         ERROR: 'error',
         EXCLUDED: 'excluded'
     }
     this.telemetry.getFingerPrint = function (cb) {
-        Fingerprint2.getV18(FPoptions, function (result, components) {
-            if (cb) cb(result, components)
-        })
+        if (localStorage.getItem('fp_deviceId')) {
+            const {result,components} = JSON.parse(localStorage.getItem('fp_deviceId'));
```
Comment (anoophm) [inline js/core/telemetryV3Interface.js]: use var and assign value to deviceDetails

File: js/core/telemetryV3Interface.js [hunk 2]
```diff
@@ -556,21 +556,27 @@ var Telemetry = (function() {
...
+            cb(result,components);
+          } else {
+            Fingerprint2.getV18(FPoptions, function (result, components) {
+              localStorage.setItem('fp_deviceId', JSON.stringify({result, components}))
```
Comment (anoophm) [inline js/core/telemetryV3Interface.js]: staore are fpDetails

**Review by anoophm — CHANGES_REQUESTED** [review]
Please do below steps allong with review changes
1. Update version in package.json and fingenrprint version. 
2. Run build command and commit those files as well
3. Do NPM publish of this library 
4. Update package version in portal and raise pr to release 2.1 branch

File: js/core/telemetryV3Interface.js (hunk 1 above)
Comment (sudip-mukherjee) [reply js/core/telemetryV3Interface.js]: done

File: js/core/telemetryV3Interface.js (hunk 2 above)
Comment (sudip-mukherjee) [reply js/core/telemetryV3Interface.js]: done

**Review by anoophm — APPROVED** [review]

**Review by rajeevsathish — APPROVED** [review]

**Review by rajeevsathish — APPROVED** [review]

**Pull request merged**